modules/vba_documents/app/controllers/vba_documents/v2/uploads_controller.rb

```diff
@@ -80,18 +80,25 @@ def download
         File.delete(zip_file_name)
       end
 
+      # rubocop:disable Metrics/MethodLength
```

Comment:
I'll be revisiting this method for a refactor fairly soon. However, I didn't have the time for a full regression just now. The method is just over our MethodLength threshold, 21 lines when the limit is 20. This code for V2 is also not live yet (behind a feature flag).